pages/project/[id].tsx

```diff
@@ -69,6 +83,21 @@ export default function ProjectPage() {
         },
     };
 
+    var member_links = [];
+    if (members !== null) {
+        for (var i = 0; i < members.length; i++) {
+            var item = members[i];
+            member_links.push(
+                <Button className={style.member_item}
+                    variant="outlined"
+                    color="info"
+                    onClick={_ => {
+                        router.push(`/profiles/${item[0]}`)
```

Comment:
If you used the type, mentioned before this could be `name` and `id` instead of `item[0]` and `item[1]`.

Reply:
yep